pages/1.8/release-notes/index.md

```diff
@@ -462,14 +462,13 @@ For more information, see the [documentation](/services/spark/).
 - Marathon and Jobs (Metronome) run as non-root
 - Switch to `/etc/os-release` for OS Detection
 - Switch to [argparse's](https://docs.python.org/3/library/argparse.html) default help for `dcos_generate_config.ee.sh`
```

Comment:
Either this needs an enterprise tag, or we should remove the enterprise bit of the genconf file name. Please check.